resources/views/inputs/switch.blade.php

```diff
@@ -35,7 +39,7 @@ class="hidden"
         @if($default ?? false) checked @endif
     />
 
-    <span class="font-semibold {{ $labelClass ?? '' }}" :class="{ 'text-theme-secondary-500': value }">
+    <span class="font-semibold {{ $labelClass ?? '' }}" :class="{ 'text-theme-secondary-500': !value }">
```

Comment:
why did this change?

Reply:
No clues of how it worked before, but was just the wrong direction when I tried it, so changed.